config/cmake_ext_mod/HDFTests.c

```diff
@@ -88,14 +88,6 @@ int main(void)
 }
 #endif
 
-#ifdef STDC_HEADERS
-#include <stdlib.h>
-#include <stdarg.h>
-#include <string.h>
-#include <float.h>
-int main() { return 0; }
-#endif /* STDC_HEADERS */
```

Comment:
This is the CMake "test" for certain C headers being present. It needs to be completely removed.